src/plugins/filecheck/README.md

```diff
@@ -0,0 +1,31 @@
+- infos = Information about the filecheck plugin is in keys below
+- infos/author = Thomas Waser <thomas.waser@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/placements = pregetstorage precommit
+- infos/description =
+
+## Introduction ##
+
+The filecheck plugin validates files testing: encoding, lineendings, BOM, printable characters and null bytes.
+
+## Configuration ##
+
+`checkLE`
+`validLE`
+When the `checkLE` key is present, the plugin checks the file for consistent line endings. If you want to validate for a specific line ending you can supply it with the `validLE` key. Valid values are: `CR`, `LF`, `CRLF`, `LFCR`.
+
+`rejectNull`
```

Comment:
avoid such camel case.

As discussed in #302 we want to always separate with `/`:
so it would be: `reject/null`